check_tool_connection_frequency: 2.0
vacuum_pump_timeout: 3.0
gripper_timeout: 3.0